internal/pkg/aws/applicationautoscaling/applicationautoscaling.go

```diff
@@ -0,0 +1,64 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package applicationautoscaling provides a client to make API requests to Application Auto Scaling.
+package applicationautoscaling
```

Comment:
omg this package name 😭 

Can we change it to the type alias that you have below? `package aas` or `appautoscaling`. I'd prefer the former with your pkg comment that nicely says it's Application Auto Scaling.